testsuite/docs-examples-cpp/run.py

```diff
@@ -13,6 +13,7 @@
 # Prep:
 command += run_app("cmake -E copy " + test_source_dir + "/../common/grid-small.exr grid.exr")
 command += run_app("cmake -E copy " + test_source_dir + "/../common/tahoe-small.tif tahoe.tif")
+command += run_app("cmake -E copy " + test_source_dir + "/../common/grid-small.exr texture_input.exr")
```

Comment:
There's already a copy of grid-small.exr, named grid.exr. Why not just use that as the input?

Reply:
No real reason, grid is just as good.
It was just to get the doc to look a certain way, but grids make good textures too.

Reply:
I'm not sure I understand this comment, Danny. Both before and after, you were using the grid-small image. But in one case, you copied it to be named "texture_input" rather than already using the identical file that was named "grid.exr".

Reply:
Yeah - I was just taking small-grid and renaming it.
It was just gymnastics so that the doc would say:
```
Input = ImageBuf("texture_input.exr")
```

But you are right, the doc is clear enough with:
```
Input = ImageBuf("grid.exr")
```